posthog/api/test/batch_exports/test_create.py

```diff
@@ -54,11 +55,17 @@ def test_create_batch_export_with_interval_schedule(client: HttpClient, interval
     client.force_login(user)
 
     with start_test_worker(temporal):
-        response = create_batch_export(
-            client,
-            team.pk,
-            batch_export_data,
-        )
+        with mock.patch(
+            "posthog.batch_exports.http.posthoganalytics.feature_enabled", return_value=True
+        ) as feature_enabled:
+            response = create_batch_export(
+                client,
+                team.pk,
+                batch_export_data,
+            )
+
+        if interval == "every 5 minutes":
+            feature_enabled.assert_called_once_with("high-frequency-batch-exports", str(team.uuid))
```

Comment:
While the feature flag exists, we check this. Once removed (as everybody has access to high frequency exports) we can delete this mock + assert.